default-plugins/status-bar/src/main.rs

```diff
@@ -230,7 +230,10 @@ impl ZellijPlugin for State {
                 println!("{}\u{1b}[48;5;{}m\u{1b}[0K", first_line, color);
             },
         }
-        println!("\u{1b}[m{}\u{1b}[0K", second_line);
+
+        if _rows > 1 {
```

Comment:
Let's change this to `rows` - it was previously prefixed with an underscore to denote that it's an unused variable in this function. Now we use it.